src/widgets/Menu/MenuLink.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { AnchorHTMLAttributes } from "react";
+import { NavLink } from "react-router-dom";
+
+const MenuLink: React.FC<AnchorHTMLAttributes<HTMLAnchorElement>> = ({ href, ...otherProps }) => {
+  const isHttpLink = href?.startsWith("http");
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const Tag: any = isHttpLink ? "a" : NavLink;
```

Comment:
The type should be `"a" | typeof NavLink`?

Reply:
tbh I can't find a way to make this work